test/verify/check-accounts

```diff
@@ -23,6 +23,7 @@ import datetime
 import parent
 from testlib import *
 
+@skipImage("Don't run upstream tests against distro packages", "rhel-7-5-distropkg", "rhel-7-6-distropkg")
```

Comment:
Seems okay as per our IRC discussion. But then please remove the existing two -distropkg special cases from the test below.

Reply:
Of course! Fixed.